src/components/countryDetails/CountryDetails.tsx

```diff
@@ -1,45 +1,214 @@
-import {
-	Link,
-	LoaderFunction,
-	useLoaderData,
-	useParams,
-} from "react-router-dom";
+import { useEffect } from "react";
```

Comment:
Please think about the composition of this component. It has way too much responsibility. Take a look at  the component and think how to split that into separate components

Reply:
I moved two columns "left" and "right" with country info to seperate components.

Reply:
Can you also move  the lists that you iterate and dynamically create to separate components?